source/crud/tests/unified/bulkWrite-deleteMany-let.yml

```diff
@@ -0,0 +1,92 @@
+description: "BulkWrite deleteMany-let"
+
+schemaVersion: "1.0"
+
+createEntities:
+  - client:
+      id: &client0 client0
+      observeEvents: [ commandStartedEvent ]
+  - database:
+      id: &database0 database0
+      client: *client0
+      databaseName: &database0Name crud-tests
+  - collection:
+      id: &collection0 collection0
+      database: *database0
+      collectionName: &collection0Name coll0
+
+initialData: &initialData
+  - collectionName: *collection0Name
+    databaseName: *database0Name
+    documents:
+      - { _id: 1 }
+      - { _id: 2 }
+
+tests:
+  - description: "BulkWrite deleteMany with let option"
+    runOnRequirements:
+      - minServerVersion: "5.0"
+    operations:
+      - object: *collection0
+        name: bulkWrite
+        arguments:
+          requests:
+            - deleteMany:
+                filter: &filter
+                  $expr:
+                    $eq: [ "$_id", "$$id" ]
+          let: &let0
+            id: 1
+    expectEvents:
+      - client: *client0
+        events:
+          - commandStartedEvent:
+              command:
+                delete: *collection0Name
+                deletes:
+                  - q: *filter
+                    limit: 0
+                let: *let0
+    outcome:
+      -
+        collectionName: *collection0Name
+        databaseName: *database0Name
+        documents:
+          - { _id: 2 }
+
+  - description: "Bulk Write deleteMany with let option unsupported
```

Comment:
I don't believe any other tests split test descriptions across multiple lines. I'd prefer if we didn't start doing this, as it could make it harder to grep test files for a name (I frequently do this when looking into test failures, since we expose these names in our test runner).